app/src/main/java/org/sagebionetworks/research/mpower/tracking/util/WhichTaskUtil.java

```diff
@@ -0,0 +1,48 @@
+package org.sagebionetworks.research.mpower.tracking.util;
+
+import android.support.annotation.NonNull;
+
+import org.sagebionetworks.research.mpower.Tasks;
+import org.sagebionetworks.research.mpower.tracking.model.TrackingStepView;
+import org.sagebionetworks.research.mpower.tracking.model.TrackingSubstepInfo;
+
+/**
+ * Util to encapsulate the logic of figuring out which task (TRIGGERS, MEDICATION, or SYMPTOMS) a TrackingStepView
+ * represents.
+ */
+public class WhichTaskUtil {
+    public static final String SYMPTOM_LOGGING_TYPE_KEY = "symptomLogging";
+    public static final String MEDICATION_REMINDERS_TYPE_KEY = "medicationReminders";
+
+    private WhichTaskUtil() {}
+
+    /**
+     * Returns the type of task that the given TrackingStepView represents.
+     * @param trackingStepView the TrackingStepView to get the type of task for.
+     * @return the type of task that the given TrackingStepView represents.
+     */
+    public static String whichTask(@NonNull TrackingStepView trackingStepView) {
```

Comment:
I'd suggest making an extension for TrackingStepView instead, much easier for a dev to discover and use.